assets/js/theme/cart.js

```diff
@@ -163,17 +179,23 @@ export default class Cart extends PageManager {
         $('.cart-remove', this.$cartContent).on('click', (event) => {
             const itemId = $(event.currentTarget).data('cart-itemid');
             const openTime = new Date();
-            const result = confirm($(event.currentTarget).data('confirm-delete'));
-            const delta = new Date() - openTime;
-            event.preventDefault();
-
-            // Delta workaround for Chrome's "prevent popup"
-            if (!result && delta > 10) {
-                return;
-            }
+            const result = swal({
+                text: 'Are you sure you want to remove this item from your cart?',
+                type: 'warning',
+                showCancelButton: true,
+                confirmButtonText: 'Yes, delete it!',
+            }).then(() => {
+                const delta = new Date() - openTime;
+
+                // Delta workaround for Chrome's "prevent popup"
+                if (!result && delta > 10) {
```

Comment:
A question. Do we need this hack anymore? It looks like `sweetalert2` is just a regular modal. Unlike `window.confirm`, the browser might not interfer.